src/coreclr/debug/di/process.cpp

```diff
@@ -11468,7 +11468,7 @@ void CordbWin32EventThread::Win32EventLoop()
         rghWaitSet[0] = m_threadControlEvent;
 
         DWORD dwWaitTimeout = INFINITE;
-
+        event = {};
```

Comment:
```suggestion
        DEBUG_EVENT event = {};
```
Move the declaration here as well?